src/js/hca-rjsf/config/form.js

```diff
@@ -233,8 +236,40 @@ const formConfig = {
         }
       }
     },
+    vaBenefits: {
+      title: 'VA Benefits',
+      pages: {
+        vaBenefits: {
+          path: 'va-benefits/basic-information',
+          title: 'Current Compensation',
```

Comment:
Oh blast. You're right. I just copied and pasted this from the old one without thinking. Good catch!